### ISSUE TITLE
Windows vanish

### ISSUE THREAD

**Issue by git-claudia** [zim-desktop-wiki/zim-desktop-wiki#2300]
I'm using version 0.75.1 on Windows 10.
Unfortunately I haven't yet been able to reproduce this bug which happened for the second time now:

In my wiki I have a link to a single page. At first I created that page as part of the wiki but moved it to a different location and created a link from the main wiki to this page. With the main wiki and the single page open on an external monitor I closed my laptop which then went to sleep. When opening it again without the additional monitor the page was gone from view. I could still see it in the Windows task view but as an empty window without the text preview. When I clicked on the lost page from there everything else was apparently displayed as background window but the chosen page didn't show.
A while ago this already happened with the help page. In this case I can't recall what I did before the window vanished, it has been too long.

Closing the windows from the task view didn't help, neither did reinstalling ZIM using the unistaller delivered with the installer package.

I added a screenshot of the task view displaying my wiki. The leftmost window is the one I can work with, the other two are the windows I can no longer see on the desktop (and which are contrary to the display not empty).

Unfortunately I can't access the debug log. The menu answers me with "No application found for: text/plain" and starting ZIM with"zim -D" from the command line just opens my wiki without additional information.

![task view](https://user-images.githubusercontent.com/123082470/214287142-68462d15-e664-4946-b4d0-63ca7d2466a2.jpg)

**Comment by jaap-karssenberg** [comment]
To access the debug log, you need to set a text editor preference in the
preferences dialogs under "applications" - this would allow the menu item
to work.

However, this very much sounds like something going wrong in the
interaction between the Windows window manager and/or Gtk . If this is the
case, the zim specific logging will not contain information on this I'm
afraid.

--- Jaap

**Comment by git-claudia** [comment]
Update:
The single page wasn't part of a wiki anymore, I had just moved the file to a different directory and linked it as an external file. To be able to still open it as a zim file I had changed Windows to open .txt files with zim (ugly, but it worked). Today I created a new zim wiki in the directory with the single page. It automatically recognized that page as part of the wiki. Now I can access the page again!
